src/test/java/teammates/test/cases/browsertests/InstructorSubmissionAdjustmentUiTest.java

```diff
@@ -39,8 +39,8 @@ protected void prepareTestData() {
         removeAndRestoreDataBundle(testData);
     }
 
-    @Test
-    public void testAdjustmentOfSubmissions() {
+
+    public void testAdjustmentOfSubsmissions() {
```

Comment:
Ok I'll look into it. I'm a bit unclear as to what changes I need to make to this InstructorSubmissionsAdjustmentUiTest.java code.